compiler/rustc_attr_parsing/src/attributes/mod.rs

```diff
@@ -33,7 +33,7 @@ pub(crate) mod transparency;
 pub(crate) mod util;
 
 type AcceptFn<T> = fn(&mut T, &AcceptContext<'_>, &ArgParser<'_>);
-type AcceptMapping<T> = &'static [(&'static [rustc_span::Symbol], AcceptFn<T>)];
+type AcceptMapping<T> = &'static [(Symbol, AcceptFn<T>)];
```

Comment:
We can't do this. There are attributes that require more than one symbol in its path, and doing this would mean we can't parse those anymore, which would be a shame. None of the first few parsers I built use this feature, but I think it is an important one

Reply:
Like namespaced attributes?

Reply:
precisely